config/locales/en.yml

```diff
@@ -26,6 +26,54 @@ en:
           label: "Yes"
         option_no:
           label: "No"
+    medical_equipment_type:
+      title: What kind of medical equipment?
+      options:
+        number_ppe:
+          label: PPE
```

Comment:
Looks like the design has been updated to spell this out as "Personal protection equipment"